### ISSUE TITLE
Clicking on EBVs should pop up explanation

### ISSUE THREAD

**Issue by jmlord** [PEM-Humboldt/projects-se-ui#94]
At booth, I saw someone in a project page click on an EBV to have the description.
Right now, nothing happens. 
A popup or explanation page would be nice.

(even though less complex, we could have the same for biological groups... what do you mean, by fish?)

**Comment by Angrypixels** [comment]
I agree for the EBVs, not so much in biological groups since is kind of obvious, unless is a generic text explaining what this means, but that could be a small subtitle.

**Comment by erikasv** [comment]
Dejar el botón de i abierto
